authorizations.md

```diff
@@ -0,0 +1,92 @@
+# Authorizations Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+An authorizations is the representation of some rights that are available to a specific user (eperson) in a defined scope, eventually the whole repository (site object).
+All the authorizations are always explicitly listed regardless to how they are grant, by direct policies via groups' membership or open to everyone (anonymous users).
+
+## Main Endpoint
+**/api/authz/authorizations**
+
+As we don't have yet an use case to iterate over all the existent authorizations the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+## Single Authorization
+**/api/authz/resourcepolicies/<:id>**
```

Comment:
There's a typo here.  This should be `/api/authz/authorizations/<:id>`

Reply:
done